book/scenarios.qmd

```diff
@@ -4,85 +4,105 @@ Seven theoretical development scenarios are defined in the first phase of the pr
+The development is located west of the city around Callerton.
 
 The new development is modelled as a combination of _open sprawl_ and _disconnected suburbia_ [signature types](https://urbangrammarai.xyz/story/#ss), combined with an estimation of the allocation of new population and a small number of jobs in retail and education.
 
 The model results in approximately:
 
 - 63 000 new residents
-- 9 250 new jobs (14% of residents)
+- 9 250 new jobs (14% of new residents)
 - no new parks
 
 
 ## Scenario 2: Mid-density mixed neighbourhood
 
-This scenario models the development in the green belt under the idea of a 15-minute neighbourhood that is dense, therefore taking up less space and mixed in terms of use. Such a neighbourhood contains not only residential housing but also a few places for new retail, commercial, and other uses. As such, it should be more self-sufficient than the low-density Scenario 1, inducing less traffic from the neighbourhood to other areas in the city. The development is assumed to be in the same area west of Callerton as in Scenario 1. The form is composed more of row houses and multi-story tenement buildings forming the centre of the new neighbourhood.
+This scenario models the development in the green belt under the idea of a 15-minute neighbourhood that is dense, therefore taking up less space and mixed in terms of use.
+Such a neighbourhood contains not only residential housing but also a few places for new retail, commercial, and other uses.
+As such, it should be more self-sufficient than the low-density Scenario 1, inducing less traffic from the neighbourhood to other areas in the city.
+The development is assumed to be in the same area west of Callerton as in Scenario 1.
+The form is composed more of row houses and multi-story tenement buildings forming the centre of the new neighbourhood.
 
-It is modelled as a combination of _accessible suburbia_, _connected residential neighbourhoods_ and _dense residential neighbourhoods_ [signature types](https://urbangrammarai.xyz/story/#ss), with an approximation of new population and job allocation. Land cover is changed accordingly to _discontinuous urban fabric_ and _continuous urban fabric_, but on a smaller area than in Scenario 1, leaving space dedicated to large urban parks.
+It is modelled as a combination of _accessible suburbia_, _connected residential neighbourhoods_ and _dense residential neighbourhoods_ [signature types](https://urbangrammarai.xyz/story/#ss), with an approximation of new population and job allocation.
+Land cover is changed accordingly to _discontinuous urban fabric_ and _continuous urban fabric_, but on a smaller area than in Scenario 1, leaving space dedicated to large urban parks.
 
 The model results in approximately:
 
 - 54 000 new residents
-- 10 500 new jobs (19 % of residents)
+- 10 500 new jobs (19% of new residents)
 - 2 300k sq.m. of new parks
```

Comment:
Does 2 300k sqm mean 2.3 million square metres? And is this standard usage? Coming from a physical sciences background, I think I would find this easier to understand if it was written as either 2.3 × 10^6 m^2, 2.3 million m^2, or 2.3 km^2. I will defer to you all, though. :)

Reply:
it is not uncommon but happy to go with million.  2.3 × 10^6 m^2 surely is not the way normal people would understand :)

Reply:
Actually, let's do `2 300 000 sq. m`, if that's alright!